easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -1234,7 +1234,11 @@ def det_full_module_name(self, ec, force_visible=False):
     def det_install_subdir(self, ec):
         """Determine name of software installation subdirectory."""
         self.log.debug("Determining software installation subdir for %s", ec)
-        subdir = self.mns.det_install_subdir(self.check_ec_type(ec))
+        if build_option('fixed_installdir_naming_scheme'):
+            self.log.debug("Using fixed naming software installation subdir for %s", ec)
```

Comment:
move this down, also mention the subdir itself in the log message? Also, `ec` is a dictionary, this will be a big/unreadable log message?

Both remarks fixed with this:

```python
self.log.debug("Using fixed naming software installation subdir: %s (ec: %s)", subdir, ec)
```